src/test/java/executor/service/ProxyConfigHolderTest.java

```diff
@@ -0,0 +1,105 @@
+package executor.service;
+
+import executor.service.model.ProxyConfigHolder;
+import executor.service.model.ProxyCredentials;
+import executor.service.model.ProxyNetworkConfig;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class ProxyConfigHolderTest {
+    private final String hostname = "hostname";
+    private final Integer port = 1234;
+
+    private final String userName = "username";
+    private final String password = "password";
+
+
+    private final ProxyNetworkConfig proxyNetworkConfig = new ProxyNetworkConfig(hostname, port);
+    private final ProxyCredentials proxyCredentials = new ProxyCredentials(userName, password);
+
+    private ProxyConfigHolder actualProxyConfigHolder;
+    private ProxyConfigHolder emptyProxyConfigHolder;
+
+    private ProxyConfigHolder expectedProxyConfigHolder;
+
+    @BeforeEach
+    void setUp() {
+        actualProxyConfigHolder = new ProxyConfigHolder(proxyNetworkConfig, proxyCredentials);
+        emptyProxyConfigHolder = new ProxyConfigHolder();
+        expectedProxyConfigHolder = new ProxyConfigHolder(proxyNetworkConfig,proxyCredentials);
+    }
+
+    @Test
+    void notNull() {
```

Comment:
let's use the same code style
```suggestion
   public void notNull() {
```